src/averageAges.js

```diff
@@ -1,61 +1,53 @@
 'use strict';
 
-/**
- * Implement calculateMenAverageAge function
- *
- * Function returns average age of men in array. If `century` is specified then
- * function calculates average age only for men who died in this century
- *
- * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
- *
- * @param {object[]} people
- * @param {number} century - optional
- *
- * @return {number}
- */
+function calculateAge(people) {
+  const ageArr = people.map((person) => person.died - person.born);
+  const sumAge = ageArr.reduce((a, b) => a + b);
+
+  return sumAge / ageArr.length;
```

Comment:
- try to do it without map here
- you should avoid names like `a` `b`
- you should not use data type in variable names